### PR TITLE
feat(frontend): dapp submit your dapp

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

Give the possibility request the adding of a dapp to the dapp explorer. For the first version, we link to github, where users can create a request.

# Changes

- Add a button to submit dapps to the dapp explorer
- Note: For now there is no template in github. This will be added in a second iteration. cc @Stefan-Berger-DFINITY 

image

### CODE REVIEW

**Comment by Stefan-Berger-DFINITY** [comment]
Template Ticket: https://dfinity.atlassian.net/browse/OISY-188

**Review by peterpeterparker — CHANGES_REQUESTED** [review]
> Note: assuming we want developers to submit Pull Request

Thanks for the PR. However, to my knowledge, this cannot work because the OISY Wallet repository does not currently accept external contributions. That said, I might be missing some context, so I'm blocking it until we clarify the following questions:

1. Are you and the stakeholders planning to make OISY Wallet accept external contributions? If yes, can you share the link to the request and confirm that security has approved this change?

2. Alternatively, are you and the stakeholders okay with leading potential contributors to a weak developer experience where their PR will be automatically closed? Also, are there plans to provide any documentation, or will devs be expected to figure things out on their own?

f the answer to all these questions is no, I assume this feature should be refined further with the stakeholders.

**Comment by peterpeterparker** [comment]
Note that an "issue" can be opened by anyone, while a "(pull) request" cannot currently.

I just want to be sure I understand the requirements as described above because depending on the expectation, the outcome can be quite different. And again, I may just have misunderstood what "Submit your dApp" means because I'm used to a certain open source flow. Thanks in advance for the feedback!

**Comment by loki344** [comment]
@peterpeterparker added template in url

File: src/frontend/src/lib/components/dapps/DappsExplorerSignedIn.svelte [hunk 1]
```diff
@@ -83,6 +83,19 @@
 	{/each}
 </ul>
 
+<a
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/dapps/DappsExplorerSignedIn.svelte:86]: In a follow-up PR, can you extract this in a specific component. `DappsExplorerSignedIn` starts to be big and this feautre can be scoped.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**